plugins.xml

```diff
@@ -12590,4 +12590,40 @@ the registration of DOIs with mEDRA.</p>]]></description>
 			<description locale="es">Este módulo envía un recordatorio a la dirección de correo electrónico del revisor cuando se le asigna un envío. El recordatorio les informa del periodo de revisión, que puede añadirse a los principales calendarios digitales.</description>
 		</release>
 	</plugin>
+	<plugin category="generic" product="downloadReviews">
+		<name locale="en">Download Reviews</name>
+		<name locale="en_US">Download Reviews</name>
+		<name locale="es_ES">Descargar reseñas</name>
```

Comment:
For 3.4.0-x (and newer), you'll need to add a duplicate line with `locale="es"` (`name`, `summary`, and `description` elements)